src/jvm/test/resources/expected_logs/internal_bug_report.txt

```diff
@@ -0,0 +1,11 @@
+It seems you've found a bug in Lincheck.
+Please report it to JetBrains.
```

Comment:
Why to JetBrains? How?

"Please submit an issue here https://github.com/JetBrains/lincheck/issues. We kindly ask providing a stack trace printed below and the code that causes the error." (please improve this text)